src/fetch/transfer_file.py

```diff
@@ -108,6 +110,16 @@ def from_dict(cls, obj: dict[str, str]) -> Transfer:
             amount_wei=int(obj["amount"]),
         )
 
+    @classmethod
+    def from_dataframe(cls, pdf: pd.DataFrame) -> Transfer:
```

Comment:
The norm naming for dataframes is usually df. pdf caught me by surprise.

```suggestion
    def from_dataframe(cls, df: pd.DataFrame) -> Transfer:
```

Reply:
df does not conform to pylint default naming conventions. Variable have to have at least three characters.